deployment/cdk/opensearch-service-migration/lib/stack-composer.ts

```diff
@@ -301,6 +335,68 @@ export class StackComposer {
             this.stacks.push(mskUtilityStack)
         }
 
+        let migrationAnalyticsStack;
+        let analyticsDomainStack;
+        if (migrationAnalyticsServiceEnabled && networkStack) {
```

Comment:
Good point. No, they could definitely be split later--a user would have to provide their own otel config, but otherwise it should be pretty seamless. And probably have to `...Enabled` params--for the otel collector and for the ES stack.